test/blackbox-tests/test-cases/enabled_if/eif-library-name-collision.t

```diff
@@ -40,6 +40,7 @@ For private libraries
   Error: Library foo is defined twice:
   - a/dune:1
   - b/dune:1
+  -> required by alias default
```

Comment:
Can't track why this line appears now by looking at the code. Do you know?

Reply:
Since the evaluation is a bit delayed, we're running into the when executing the rule rather than loading them. It's not an issue. It's usually an issue when we do the opposite, and raise too eagerly.